README.md

```diff
@@ -1843,7 +1851,18 @@ Robotics: Science and Systems (RSS), 2020. [PDF](http://www.roboticsproceedings.
 IEEE/RSJ Int. Conf. Intelligent Robots and Systems (IROS), 2020.
 
 
-## 3D hand pose estimation
+<a name="pose_estimation"></a>
+## Pose Estimation
```

Comment:
Thanks. Could we call it "Object Pose Estimation" ?
I may be biased by my background, but when I read "pose estimation" I think of "(camera) pose estimation"